ntbs-service/Pages/Notifications/Edit/ClinicalDetails.cshtml.cs

```diff
@@ -233,7 +237,7 @@ public ContentResult OnGetValidateNotificationSiteProperty(string key, string va
 
         public ContentResult OnGetValidateClinicalDetailsYearComparison(int newYear, int existingYear)
         {
-            return ValidationService.ValidateYearComparison(newYear, existingYear);
+            return ValidationService.ValidateYearComparison(newYear, existingYear, "BCG vaccination year");
```

Comment:
This has become specific for one field on this page - I've done this in a few places which feels like bad practice but there are no pages where these types of comparisons are done on multiple fields so it works. I'm wondering if the best solution is to make the year-comparison component take a prop of what field it's validating from the html and supply it to this function or contain more of that logic in the component itself so this function never knows what property it is validating. Any thoughts?